release/nightly_tests/wait_cluster.py

```diff
@@ -26,15 +26,6 @@
 next_feedback = start
 max_time = start + args.max_time_s
 
-
-def num_alive_nodes():
```

Comment:
Looks like `wait_cluster` and `_wait_cluster` is somehow linked?

Reply:
If you add `wait_for_nodes` in the test run configuration, the infra can wait until the nodes are up and you don't need to do this in the script anymore

Reply:
But if this is dynamic you may want to keep it like it is

Reply:
Oh the weird thing is I didn't change this file, but it was changed for some reasons like this..